TrafficCapture/dockerSolution/src/main/docker/migrationConsole/lib/console_link/console_link/cli.py

```diff
@@ -50,10 +50,8 @@ def cli(ctx, config_file, json, verbose):
 @cli.group(name="clusters", help="Commands to interact with source and target clusters")
 @click.pass_obj
 def cluster_group(ctx):
-    if ctx.env.source_cluster is None:
-        raise click.UsageError("Source cluster is not set")
-    if ctx.env.target_cluster is None:
-        raise click.UsageError("Target cluster is not set")
+    if ctx.env.source_cluster is None and ctx.env.target_cluster is None:
+        raise click.UsageError("Neither source nor target cluster is defined.")
```

Comment:
Non-blocking grammar nit: `is` => `are`

Reply:
equally nitpicky, but `is` is technically right because both are singular nouns (but both versions are common) (https://www.cjr.org/language_corner/either_wins.php)